app/views/documents/history/_whitehall_entry.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="app-timeline-entry">
+  <% if entry.details.imported_from_whitehall? %>
+    <%= render "govuk_publishing_components/components/notice", {
+      title: "#{t "documents.history.entry_types.whitehall_migration.#{entry.details.entry_type}"}",
+      description_text: format_date(entry.created_at)
+    } %>
+  <% else %>
+    <h4 class="govuk-heading-s">
+      <%= t "documents.history.entry_types.whitehall_migration.#{entry.details.entry_type}" %>
```

Comment:
...and here?

Reply:
I didn't use the braces here, as I understood your suggestion, as this would make this `t` call inconsistent with our regular approach of using no braces.